scipy/optimize/tests/test_optimize.py

```diff
@@ -1259,6 +1259,26 @@ def dfunc(z):
             assert func(sol1.x) < func(sol2.x), \
                    f"{method}: {func(sol1.x)} vs. {func(sol2.x)}"
 
+    @pytest.mark.parametrize(
+        'bounds', [None, [[0.0, 0.0], [-np.inf, +np.inf], [-np.inf, +np.inf]]],
+    )
+    @pytest.mark.parametrize('method', ['l-bfgs-b'])
```

Comment:
Since, we are testing only for one method here, may be we don't need `method` as input argument to `test_minimize_callback_result`. We can directly pass, `l-bfgs-b` to `optimize.minimize` call below.